### DISCUSSION TITLE
Welcome to sliitfoss Discussions!

### DISCUSSION THREAD

**Discussion by chamodshehanka** [sliit-foss/sliitfoss#106] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chamodshehanka** [comment, marked as answer]
@sajeetharan Can you go through our code base and suggest things to refactor. I think we almost done (MVP release)

**Reply by Safnaj** [reply]
I just came here to explore GitHub Discussions

**Reply by chamodshehanka** [reply]
This feature is amazing 😻.Let's have a discussion
